.github/workflows/pkgci_test_sharktank.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         include:
           # CPU
           - name: cpu_llvm_task
-            runs-on: ubuntu-20.04
+            runs-on: nodai-amdgpu-w7900-x86-64
```

Comment:
Add a matrix entry for GPU using the self-hosted runner instead. CPU tests should continue to run on GitHub-hosted runners for reliability/availability/cost.

Here's an example: https://github.com/iree-org/iree/blob/a30a4193bd58e7562e6b8a730e6f1c1962a71366/.github/workflows/pkgci_test_onnx.yml#L26-L42